pkg/kafka/auth/config.go

```diff
@@ -39,6 +39,7 @@ var authTypes = []string{
 // AuthenticationConfig describes the configuration properties needed authenticate with kafka cluster
 type AuthenticationConfig struct {
 	Authentication string
+	TlsEnabled	   bool
```

Comment:
we should not need this once #1838 is finished

Reply:
makes sense. Reverted the change. I looked at the PR #1838  looks good. However, will take a closer look over the weekend.